hostfactory/host_provider/invoke_provider.sh

```diff
@@ -12,14 +12,14 @@ export PYTHONPATH=$PYTHONPATH:$scriptDir/src
 env > /tmp/invoke.env
 
 embedded_python=/opt/cycle/jetpack/system/embedded/bin/python
-venv_path=/opt/ibm/spectrumcomputing/hostfactory/1.2/providerplugins/azurecc/venv/bin
+venv_path=$HF_TOP/$HF_VERSION/providerplugins/azurecc/venv/bin
 
 
 if [ -e $venv_path ]; then
    # Check group membership
 	touch /opt/cycle/jetpack/logs/jetpack.log 1>&2 2> /dev/null
 	
-	if [ $? == 0 ]; then
+	if [ $? != 0 ]; then
```

Comment:
Can we simply eliminate writing to jetpack.log?

Reply:
i.e. I don't think there should be any path where we write, from the provider, to the jetpack.log. If possible.

Reply:
yeah we can eliminate this. I think its to check group membership? but can test it

Reply:
I think we only need to check group membership if they're trying to use the jetpack python.   
I'd prefer to remove the group membership check if we can